packages/terra-clinical-item-view/src/ItemView.jsx

```diff
@@ -25,48 +25,68 @@ const propTypes = {
    */
   accessoryAlignment: PropTypes.oneOf(['alignTop', 'alignCenter']),
   /**
+   * The height and width value the accessory should receive. Maximum scalar is 60px.
+   */
+  accessoryScale: PropTypes.string,
+  /**
    * The react element to be placed in the start aligned accessory position.
    */
-  startAccessory: PropTypes.element,
+  startAccessory: PropTypes.node,
+  /**
+   * Indicates whether or not space is allocated for the start accessory if none is given.
+   */
+  alignStartAccessory: PropTypes.bool,
   /**
    * The react element to be placed in the end aligned accessory position.
    */
-  endAccessory: PropTypes.element,
+  endAccessory: PropTypes.node,
+  /**
+   * Indicates whether or not space is allocated for the end accessory if none is given.
+   */
+  alignEndAccessory: PropTypes.bool,
   /**
    * An array of react display elements to be presented.
    */
   displays: PropTypes.arrayOf(PropTypes.element),
   /**
    * The react element for the comment element.
    */
-  comment: PropTypes.element,
+  comment: PropTypes.node,
 };
 
 const defaultProps = {
   layout: 'oneColumn',
   textEmphasis: 'default',
   isTruncated: false,
   accessoryAlignment: 'alignCenter',
+  accessoryScale: '1em',
   startAccessory: undefined,
+  alignStartAccessory: false,
   endAccessory: undefined,
+  alignEndAccessory: false,
   displays: [],
   comment: undefined,
 };
 
 class ItemView extends React.Component {
 
-  static renderAccessory(accessory, accessoryAlignment) {
+  static renderAccessory(alignAccessory, accessory, accessoryAlignment, accessoryScale) {
     let accessorySection;
-    if (accessory) {
+    if (accessory || alignAccessory) {
       const accessoryClassNames = cx(
         'accessory',
         { 'accessory-align-center': accessoryAlignment === 'alignCenter' },
         { 'accessory-align-top': accessoryAlignment === 'alignTop' },
       );
 
+      let scaledAccessory = accessory;
+      if (accessory && accessoryScale !== null) {
+        scaledAccessory = React.cloneElement(accessory, { width: accessoryScale, height: accessoryScale });
```

Comment:
This will spread width and height directly onto any accessory component. If the component isn't an Icon it will throw warning / errors on the dom for invalid props and override valid props if they are given.

Should the props be set directly on the icon accessory rather than passed through the ItemView to override the props on the Icon?

Reply:
Merging the existing props with the new props of `width` and `height` doesn't actually thrown warnings or errors on the dom for invalid props. React treats these as style props and will apply them as so. 

We could handle setting the accessory width, which is done [here](https://github.com/cerner/terra-clinical/pull/222/files#diff-8788aebd203a6b2eaee49ee3dab4e2e6R88).

The reason we have this logic scaling logic comes from these two issues logged a while back #114 & #115. The maximum height & width keeps the Icon scaled and positioned correctly within the space provided by the accessory when font-size scaled up.

Reply:
It does look like these are special props that are silently spread, but I tested spreading these onto React nodes and it didn't actually affect the styled width or height of the Component, are you able to see changes in width in height on any component other than Icon?

For the scaling, the issues suggest the icons should scale by only adjusting the font-size.  Should these Icons work similar to that of the List chevron which will adjust only by setting the font-size?

Reply:
hmmm I see your point regarding the scaling & it being an icon specific implementation. & Yes I do think icons should only be adjusting to font-size. I did some more testing, it looks like you are right and the height and with are only being honored by the Icon. and when applying it as a style. I'll go ahead and remove lines 82-86 and leave the accessory sizing to the consumer.

Then, I am thinking update the prop to be `accessoryWidth` that is applied. The ItemView's maxWidth keeps the width with the 60px maxWidth.